insta/app/controllers/home_controller.rb

```diff
@@ -0,0 +1,12 @@
+class HomeController < ApplicationController
+
+  def index
+    if current_user
+      @posts = Post.order(created_at: :desc).page(params[:page]).per(5) #@Kaminari GEM pagination
```

Comment:
I would love to see how this works but I can't find the route you use to show it. 
home/index just shows a form to create a new post